src/VirtoCommerce.Platform.Web/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -107,5 +110,30 @@ public static IServiceCollection AddSecurityServices(this IServiceCollection ser
 
             return services;
         }
+
+
+
+        public static IServiceCollection AddLibraries(this IServiceCollection services)
```

Comment:
I would like to recommend move this logic into the new class `UnmanagedLibraryLoader` and encapsulate all this logic there.